### DISCUSSION TITLE
My github account has been blocked, I want to use this email, can not unassociate email. Contact support Select mobile number No China

### DISCUSSION THREAD

**Discussion by Damon-Suan** [community/community#147327] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My github account has been blocked, I want to use this email, can not unassociate email. Contact support Select mobile number No China

**Comment by ghost** [comment]
Me too.

My account  https://github.com/warmthsea was banned for unknown reasons

**Comment by shahadat26** [comment]
My github account https://github.com/suspended was suspended. Show this message "Access to your account has been suspended due to a violation of our [Terms of Service](https://docs.github.com/articles/github-terms-of-service).
Please [contact support](https://support.github.com/contact/reinstatement) for more information."

**Comment by mecodeatlas** [comment]
Hi @Damon-Suan!

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion. We appreciate your understanding!
